modules/disk-uploader/pkg/parse/clioptions_test.go

```diff
@@ -0,0 +1,81 @@
+package parse_test
+
+import (
+	"go.uber.org/zap/zapcore"
+
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
Same

Reply:
I'm not sure how to format it, I have basic golang's formatter executed on this file.